frontend/src/App.tsx

```diff
@@ -11,7 +11,7 @@ import CategoryPage from "./pages/CategoryPage";
 function App() {
     return (
         <div className="App">
-            <BrowserRouter>
+            <HashRouter>
```

Comment:
Why are using HashRouter?

Reply:
I needed to change it due to the backend routing problem I had yesterday